ethstaker_deposit/deposit.py

```diff
@@ -94,7 +95,11 @@ def cli(ctx: click.Context, language: str, non_interactive: bool, ignore_connect
 def run() -> None:
     freeze_support()  # Needed when running under Windows in a frozen bundle
     check_python_version()
-    cli()
+
+    try:
+        cli()
+    except (ValueError, ValidationError) as e:
+        click.echo(f"\nError: {e}\n")
 
```

Comment:
A good convention here would be to exit with an exit code that isn't 0 so that automation can be easier for different tools. Calling `sys.exit(1)` is likely a good way to do this. See https://docs.python.org/3/library/sys.html#sys.exit

Reply:
Good catch. ack